src/transport-manager.js

```diff
@@ -212,4 +218,11 @@ class TransportManager {
   }
 }
 
+const FAULT_TOLERANCE = {
```

Comment:
We should add some jsdocs to describe each of these as they're not specified anywhere.

Reply:
I did not find any standard to document the Enum values. I added them in the main body description for now. If we increase this list, we should rethink this.